server/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -72,23 +83,62 @@ public Result dashboard(Http.Request request, Optional<String> search, Optional<
     if (trustedIntermediaryGroup.isEmpty()) {
       return notFound();
     }
+    SearchParameters searchParameters =
+        SearchParameters.builder().setNameQuery(nameQuery).setSearchDate(searchDate).build();
+
     ImmutableList<Account> managedAccounts =
-        trustedIntermediaryGroup.get().getManagedAccounts(search);
+        tiService.getManagedAccounts(searchParameters, trustedIntermediaryGroup.get());
     PaginationInfo<Account> pageInfo =
         PaginationInfo.paginate(managedAccounts, PAGE_SIZE, page.get());
-
     return ok(
         tiDashboardView.render(
             trustedIntermediaryGroup.get(),
             civiformProfile.get().getApplicant().join().getApplicantData().getApplicantName(),
             pageInfo.getPageItems(),
             pageInfo.getPageCount(),
             pageInfo.getPage(),
-            search,
+            searchParameters,
             request,
             messagesApi.preferred(request)));
   }
 
+  @Secure(authorizers = Authorizers.Labels.TI)
+  public Result updateDateOfBirth(Long accountId, Http.Request request) {
+    Optional<CiviFormProfile> civiformProfile = profileUtils.currentUserProfile(request);
+
+    if (civiformProfile.isEmpty()) {
+      return unauthorized();
+    }
+
+    Optional<TrustedIntermediaryGroup> trustedIntermediaryGroup =
+        userRepository.getTrustedIntermediaryGroup(civiformProfile.get());
+    if (trustedIntermediaryGroup.isEmpty()) {
+      return notFound();
+    }
+
+    Form<UpdateApplicantDob> form =
+        formFactory.form(UpdateApplicantDob.class).bindFromRequest(request);
+    try {
+      tiService.updateApplicantDateOfBirth(accountId, form);
+
+    } catch (IncorrectDateFormatException
+        | MissingDateOfBirthException
+        | ApplicantNotFoundException
+        | FormHasErrorException
+        | DateOfBirthNotInPastException e) {
+      return redirectToDashboardWithUpdateDateOfBirthError(e.getLocalizedMessage());
+    }
+
+    return redirect(
+        routes.TrustedIntermediaryController.dashboard(
+            /* paramName=  nameQuery */
+            Optional.empty(),
+            /* paramName=  searchDate */
+            Optional.empty(),
+            /* paramName=  page */
+            Optional.empty()));
```

Comment:
suggestion (blocking): 
```suggestion
            /* nameQuery=  */
            Optional.empty(),
            /* searchDate=  */
            Optional.empty(),
            /* page= */
            Optional.empty()));
```